okhttp/src/main/kotlin/okhttp3/RequestBody.kt

```diff
@@ -15,15 +15,17 @@
  */
 package okhttp3
 
-import java.io.File
-import java.io.IOException
-import java.nio.charset.Charset
-import kotlin.text.Charsets.UTF_8
 import okhttp3.MediaType.Companion.toMediaTypeOrNull
 import okhttp3.internal.checkOffsetAndCount
 import okio.BufferedSink
 import okio.ByteString
 import okio.source
+import java.io.File
+import java.io.FileDescriptor
+import java.io.FileInputStream
+import java.io.IOException
+import java.nio.charset.Charset
+import kotlin.text.Charsets.UTF_8
```

Comment:
Seems like we need spotless up in here enforcing import order.

Reply:
+1 

I bounce around various projects and computers at times with different conventions so I'm much more in favour of getting ktfmt and .editorconfig with IntelliJ just working such that we don't have these format wars.  If you find a perfect setup on some project, let me know otherwise I'll try to find some time to work through the sort of setup.

I suspect Square folk get to a stable point with java-code-styles but for me this is a constant annoyance.

Reply:
I’m hoping we standardize on a very short `.editorconfig`.